ddtrace/contrib/trace_utils.py

```diff
@@ -682,3 +683,17 @@ def _convert_to_string(attr):
         else:
             return ensure_text(attr)
     return attr
+
+
+def _get_trace_details_for_log_injection(span):
+    # gets correct trace id and span id format for log injection
+    # Note: logs only supports 32 length hex representation of trace_id for 128 bit ids, no base10 int support
+    trace_id = None
+    span_id = None
+    if span:
+        span_id = span.span_id
+        trace_id = span.trace_id
+        if trace_id and isinstance(trace_id, int) and trace_id > MAX_UINT_64BITS:
+            trace_id = "{:032x}".format(trace_id)
+
+    return trace_id, span_id
```

Comment:
We can simplify this to the following. This will allow us to clean up some of the default values in our log integrations:
```suggestion
    if not span:
        return "0", "0"
    elif span.trace_id > MAX_UINT_64BITS and config._128_bit_trace_id_logging_enabled:
        return "{:032x}".format(span.trace_id), str(span.span_id)
    return str(span.trace_id), str(span.span_id)
```

Reply:
removed all this